src/patternfly/components/FormControl/form-control.scss

```
@@ -81,7 +81,6 @@ $pf-c-form-control--m-search--Coordinates: "M505 442.7L405.3 343c-4.5-4.5-10.6-7
   --pf-c-form-control--m-search--BackgroundPosition: var(--pf-c-form-control--PaddingLeft);
```

What you have here works. I wonder if this background position should be setup like the others where the non-icon side horizontal padding is used to position the icon. This would also allow you to redefine `--pf-c-form-control--PaddingLeft` later in `&.pf-m-search` instead of having to declare `padding-left`

```suggestion
  --pf-c-form-control--m-search--BackgroundPosition: var(--pf-c-form-control--PaddingRight);
```